xtokens/Cargo.toml

```diff
@@ -20,23 +20,23 @@ frame-system = { git = "https://github.com/paritytech/substrate", branch = "polk
 cumulus-primitives-core = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.4", default-features = false }
 
 xcm = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.4", default-features = false }
+xcm-executor = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.4", default-features = false }
 
 orml-xcm-support = { path = "../xcm-support", default-features = false }
 orml-traits = { path = "../traits", default-features = false}
 
 [dev-dependencies]
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4" }
-polkadot-core-primitives = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.4" }
-polkadot-parachain = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.4" }
-pallet-balances = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4" }
-xcm-simulator = { git = "https://github.com/shaunxw/xcm-simulator", branch = "master" }
-cumulus-pallet-xcm-handler = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.4" }
-parachain-info = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.4" }
-xcm-executor = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.4" }
-xcm-builder = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.4" }
-
-orml-tokens = { path = "../tokens", version = "0.4.1-dev" }
-orml-traits = { path = "../traits", version = "0.4.1-dev" }
+# sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4" }
```

Comment:
why this?

Reply:
I commented out all unit tests related things and will bring them back in another PR, after `xcm-simulator` is updated.